src/linear_samplers/block_row_FJLT.jl

```diff
@@ -0,0 +1,92 @@
+"""
+    LinSysBlockRowFJLT <: LinSysBlkRowSampler
+
+A mutable structure with fields to handle FJLT row sketching. For this procedure,
+the hadamard transform and random sign swaps are applied once, then that matrix is repeatably
+sampled.
```

Comment:
The embedding matrix should be defined and the components which are kept fixed should be specified, and the component which is independently generated at each `sample` call should be specified.

Reply:
This whole matrix is now regenerated at every iteration. In v0.2 we should determine some testing to determine if it really necessary from an embedding efficiency standpoint to keep regenerating the  rademacher vector.